images/kubevirt-infra-prow/publish_multiarch_prow_images.sh

```
@@ -0,0 +1,128 @@
+#!/bin/bash -ex
+
+BAZEL_VERSION=3.0.0
+PROW_COMMIT=9f5410055c
+registry=quay.io
+repo=kubevirtci
+archs=(arm64 ppc64le)
+binaries=(clonerefs initupload entrypoint sidecar)
+
+main() {
+    local build_only
+    local tag
+    while getopts "bh" opt; do
+        case "$opt" in
+            b)
+                build_only=true
+                ;;
+            h)
+                help
+                exit 0
+                ;;
+            *)
+                echo "Invalid argument: $opt"
+                help
+                exit 1
+        esac
+    done
+    shift $((OPTIND-1))
+    tag=$(get_image_tag)
+    build_binaries
+    build_images
+    [[ $build_only ]] && return
+    push_images
+    push_manifest
+}
+
+help() {
+    cat <<EOF
+    Usage:
+        ./publish_multiarch_prow_images.sh [OPTIONS]
+
+    Build and publish multiarch prow utility images.
+
+    OPTIONS
+        -h  Show this help message and exit.
+        -b  Only build the image and exit. Do not publish the built image.
+EOF
+}
+
+get_base_image() {
+    if [ $1 == "arm64" ]; then
+        echo "arm64v8/alpine"
+    elif [ $1 == "ppc64le" ] ; then
+        echo "ppc64le/alpine"
+    else
+        echo "unsupport arch"
+        exit
```

Do we want to silently ignore this?